### DISCUSSION TITLE
Koordinaten kopieren vereinfachen

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/webgis-community#123] [Ideas]
Uns wurde folgender Userwunsch zugetragen:

Im Moment muss das Koordinatenpaar einzeln kopieren:
![image](https://github.com/gis-eni/webgis-forum/assets/127403099/a75520ec-6b23-4ab6-a3ff-77ec0c7ad2aa)

Wäre es möglich in Zukunft beide Koordinaten auf einmal zu kopieren?
![image](https://github.com/gis-eni/webgis-forum/assets/127403099/265c44c8-9261-4911-ada1-7b064a477f6a)

Zitat des Users:
_"Wenn ich die Daten in ein Dokument übernehmen will brauche ich immer x und y. Selbst wenn ich die Koordinaten im Zielprogramm (zB in Excel) in zwei Spalten einfügen will bin ich schneller, wenn ich zerst x,y in spalte 1 einfüge, dann den y-Teil aus dieser Spalte in die Zwischenablage nehme und in Spalte 2 einfüge."_

Was spricht dagegen?

**Comment by gis-eni** [comment]
Ich find die Idee grundsätzlich gut. Allerdings kann es natürlich sein (kommt sogar bei uns so vor), dass mit man die Werte einzeln braucht, um sie ein Formular zu kopieren, wo man X Y als Felder hat: Beispiel WebGIS Koordinaten Eingabe. Oder ArcMap auf Koordinate springen, etc. 
Da ist es dann wieder blöd, wenn man gleich beide Werte in der Zwischenablage hat und in ein Eingabefeld (X) einträgt:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/62c0289c-975e-4745-b5e0-fed68ac9c531)

Ich hab jetzt folgend Kompromiss implementiert:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/5bd340f7-2159-4353-bbc4-8a840e6e6d3e)

Die hellgrauen Buttons haben als ToolTip "Wert in die Zwischenablage kopieren" und funktionieren wie bisher. 
Der hervorgehobene Button hat als ToolTip: "Zeile in die Zwischenablage kopieren" und kopiert alle Werte auf einmal

Bei Zeile kopieren werden die einzelnen Zellentexte mit Tabulator (\t) miteinander verknüpft. Das bewirkt dann, dass im Excel das beim 
Einfügen auch wirklich auf zwei Zellen aufgeteilt wird:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/42c63719-bbd6-471f-b55c-b78a2c0a3e82)

Mit einem anderem Trennzeichen hätte man sonst beide Werte in einer Zelle stehen, was meiner Meinung nicht sinnvoll wäre ;)

**Reply by ue-black** [reply]
Perfekt!
